posts/2025-06-27-bioc-to-galaxy/index.qmd

```diff
@@ -0,0 +1,154 @@
+
+In this post, we spotlight one of the most exciting sessions from the event: **"Streamlining Bioinformatics: Adding Bioconductor Tools in Galaxy"**, presented by [Fabio Cumbo](https://github.com/cumbof), [Jayadev Joshi](https://github.com/jaidevjoshi83), [Bryan Raubenolt](https://github.com/thepineapplepirate), and [Daniel Blankenberg](https://github.com/blankenberg). The session explored how developers can integrate Bioconductor tools into Galaxy by wrapping them within the web-based platform to expand accessibility and usability for researchers.
+
+In case you missed the live session, don't worry-- you can access the full set of training materials, including slides and examples on their [GitHub repository](https://github.com/cumbof/gbcc25_training_session).
+
+![](bioc-galaxy-image3.jpeg){fig-alt="Github linked here" fig-align="center" width="100%"}
+
+## Choosing Galaxy
+
+[Galaxy](https://usegalaxy.org/) is an open source, web-based platform with a graphical user interface (GUI) designed to make complex scientific computational analysis more accessible, reproducible, and transparent.
+
+At its core, Galaxy is a powerful workflow system that empowers users to run bioinformatics tools and pipelines without needing advanced programming or computational training nor high-powered computing clusters.
+
+This becomes particularly valuable for users who lack computing access and/or experience as Galaxy will help you upload and store your data, use tools and pipelines to develop reproducible workflows and answer your most complex scientific inquiry.
+
+![](bioc-galaxy-image2.jpeg){fig-alt="Galaxy Home Page" fig-align="center" width="100%"}
+
+## Using Bioconductor tools in Galaxy
+
+While the conference highlighted the exciting potential of bridging together two of the most powerful open-source platforms in bioinformatics, there are other compelling reasons why this integration goes beyond a technical collaboration.
+
+Bioconductor is a powerful ecosystem of R-based tools that allow for a wide range of analyses, from differential gene expression to advanced pathway enrichment. However, many of these tools remain underutilized, especially outside computational biology, simply because not all researchers are trained in programming nor have the computing access to run such analyses.
+
+**Three key benefits for using Bioconductor tools in Galaxy are:**
+
+1)  [**Benefiting researchers**]{.underline} with increased accessibility for researchers seeking more complex analyses without coding knowledge
+
+2)  [**Benefiting developers**]{.underline} with increased visibility, recognition, and usage for developers as they expand the reach of their work
+
+3)  [**Benefiting Galaxy and Bioconductor**]{.underline} by enabling Galaxy's core values (i.e., accessibility, reproducibility, & transparency) to reach the analysis complexity of Bioconductor
```

Comment:
```suggestion
3)  [**Benefiting Galaxy and Bioconductor**]{.underline} by integrating Galaxy’s accessible, reproducible workflows with Bioconductor’s analytical power.
```

Reply:
I think we should change that sentence to something like that suggestion, as Bioconductor also enables reproducibility, & transparency.